cmd/fission-bundle/main.go

```diff
@@ -78,14 +78,26 @@ func runMessageQueueMgr(logger *zap.Logger, routerUrl string) {
 	}
 }
 
-func runStorageSvc(logger *zap.Logger, port int, filePath string) {
```

Comment:
I would still keep the name as `runStorageSvc` - weather it does use a volume or S3 - is an implementation-level detail that need not leak out/known to the caller.